README.md

```diff
@@ -79,11 +79,17 @@ Clone this repository to your desired folder: (https://github.com/HERMON-1995/Le
   git clone [https://github.com/HERMON-1995/Leaderboard-list-app-project.git]
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/223366346-dd748f52-7aaa-4767-ae85-6412e79ceeaa.png)

**[Optional]**

- [ ] Feature means what your application does for you. Application Feature has nothing to do with what you are doing in Github so it would be nice if you put some sensible features such as those shown below to make your readme file descriptive :+1:

* Implement API for Data Storage